### PR TITLE
Polish README to add homepage link

### PR DESCRIPTION
**Pull request by panpan0000**
to separate "User" guide and "developer" guide, learning from k8s README style

#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Document polishment

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]

**Review by wzshiming — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,31 +22,43 @@ Below lists the status of each release image and package and you can click any t
...
+
+- [CVEs that were found and fixed recently](https://www.cvedetails.com/vulnerability-list/vendor_id-15867/product_id-34016/Kubernetes-Kubernetes.html)
+
+- [Kubernetes-related CVEs that were found and listed on cve.org recently](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=Kubernetes)
+
+
+
+### Documents
+
+- [Homepage](https://klts.io/docs/)
```
Comment (kerthcet) [inline README.md:37]: [Documents]
[Installation] better?

Comment (panpan0000) [reply README.md]: change `homepage` to `Portal and Docs`

**Comment by panpan0000** [comment]
> /hold
thanks for review.
please help to check my update .thx.

**Comment by kerthcet** [comment]
LGTM and squash the commits pls.

**Pull request merged**